SPEC/DHT.md

```diff
@@ -39,10 +39,13 @@ A great source of [examples][] can be found in the tests for this API.
 
 ##### `Go` **WIP**
 
-##### `JavaScript` - ipfs.dht.findprovs(hash, [callback])
+##### `JavaScript` - ipfs.dht.findprovs(hash, [options, callback])
```

Comment:
Nitpick - could this please be changed to `ipfs.dht.findprovs(hash, [options], [callback])`? The other way makes it look like an array which I think is a bit confusing!

Reply:
Of course! I honestly wasn't sure what the right syntax expression would be here. Will update accordingly